frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -44,6 +38,28 @@ const handleOCSError = (res, flags, dispatch, cb) => {
   }
 };
 
+export const detectOCS: FeatureDetector = async (dispatch) => {
+  try {
+    await k8sGet(OCSServiceModel, OCS_CONVERGED_CR_NAME, CEPH_STORAGE_NAMESPACE);
```

Comment:
you could do both calls in parallel, this way the second k8sGet waits for the first one to finish

Reply:
This can be done as a follow-up :smiley: 

@bipuladh Consider using [`Promise.all`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all) to kick off & handle multiple async operations in parallel.